core/src/main/java/io/grpc/internal/Channelz.java

```diff
@@ -468,9 +468,9 @@ public SocketStats(
         SocketOptions socketOptions,
         Security security) {
       this.data = data;
-      this.local = local;
+      this.local = Preconditions.checkNotNull(local);
```

Comment:
consider adding the variable name as a second arg

Reply:
Added description